fastly/block_fastly_service_logging_blobstorage_test.go

```diff
@@ -58,15 +59,16 @@ func TestResourceFastlyFlattenBlobStorage(t *testing.T) {
 					"response_condition": "error_response",
 					"file_max_bytes":     1048576,
 					"compression_codec":  "zstd",
+					"processing_region":  "eu",
 				},
 			},
 		},
 	}
 
 	for _, c := range cases {
 		out := flattenBlobStorages(c.remote, nil)
-		if !reflect.DeepEqual(out, c.local) {
-			t.Fatalf("Error matching:\nexpected: %#v\ngot: %#v", c.local, out)
+		if diff := cmp.Diff(out, c.local); diff != "" {
```

Comment:
This was changed while debugging these tests; they were the only ones that produced useless 'diff' output, unlike all of the others.